static/app/views/alerts/rules/metric/details/metricActivity.tsx

```diff
@@ -77,6 +77,7 @@ function MetricAlertActivity({organization, incident}: MetricAlertActivityProps)
               }/releases/${encodeURIComponent(activation.activator)}/`,
               query: {project: project},
             }}
+            style={{textOverflow: 'ellipsis', overflowX: 'inherit'}}
```

Comment:
oops. this is an unrelated change to make the alert row slightly nicer.

shouldn't have any effect on the rest of the PR...

I can break this out if anyone feels strongly